src/ga4gh/vrs/_internal/models.py

```diff
@@ -329,8 +498,7 @@ class SequenceLocation(BaseModel):
         description='The end coordinate or range of the SequenceLocation. The minimum value of this coordinate or range is 0. MUST represent a coordinate or range greater than the value of `start`.',
     )
 
-    class ga4gh:
-        identifiable = True
+    class ga4gh(_Ga4ghIdentifiableObject.ga4gh):
```

Comment:
Originally I put these properties in a nested class in order for them to not be seen as related to the top level pydantic properties but still be declared on and accessible on the class. This change seems fine to share some of the structure of the `.ga4gh` nested classes, but if we can think of a cleaner way that'd also be nice.

Reply:
Hmmmm @toneillbroad do you have any suggestions? If not, we can make a separate issue for this and come back to it

Reply:
The structures as represented in this PR are good by me.